src/desktop/apps/order2/server.tsx

```diff
@@ -0,0 +1,62 @@
+import { renderLayout } from '@artsy/stitch'
+import express from 'express'
+import React from 'react'
+import { routes } from 'reaction/Apps/Order/routes'
+
+import { buildServerApp } from 'reaction/Router'
+import styled from 'styled-components'
+
+const app = (module.exports = express())
+
+app.get('/order2/:orderID*', async (req, res, next) => {
+  try {
+    const user = (req as any).user && (req as any).user.toJSON()
```

Comment:
These `(req as any)`s are obviously not ideal but it seems that they are either not typed correctly or not typed at all? I'm not sure how deep to go with this...